cypress/defs.bzl

```diff
@@ -12,6 +12,30 @@ _cypress_test = rule(
     toolchains = js_binary_lib.toolchains + ["@aspect_rules_cypress//cypress:toolchain_type"],
 )
 
+def _cypress_test_macro(name, entry_point, cypress, **kwargs):
+    _cypress_test(
+        name = name,
+        entry_point = entry_point,
+        data = kwargs.pop("data", []) + [
+            cypress,
+        ],
+        chdir = native.package_name(),
+        enable_runfiles = select({
+            "@aspect_rules_js//js/private:enable_runfiles": True,
+            "//conditions:default": False,
+        }),
+        unresolved_symlinks_enabled = select({
+            "@aspect_rules_js//js/private:experimental_allow_unresolved_symlinks": True,
+            "//conditions:default": False,
+        }),
+        # Turn off sandboxing by default to allow electron to perform write
```

Comment:
What type of writing does it do? Is not now going to be modifying files in the bindir? Or runfiles?

Reply:
No, just files in the electron `user data` folder. Cypress doesn't expose us the electron hook we need to customize this folder from its defaults

```
appData Per-user application data directory, which by default points to:
%APPDATA% on Windows
$XDG_CONFIG_HOME or ~/.config on Linux
~/Library/Application Support on macOS
```
See https://www.electronjs.org/docs/latest/api/app

We set `XDG_CONFIG_HOME` on linux but on Mac we can't fix the sandboxing violation at this time without upstream changes to cypress

Reply:
Just an idea but did you try setting `HOME`? Maybe you can make it write to `/some/safe/path/Library/Application Support`.

Reply:
Yeah, the existing rule sets home already. Unfortunately, the electron process seems to ignore this

Reply:
Can you add to the comment an example error message that occurs when sandboxed with newer versions of cypress?